src/csvcubed/utils/sparql_handler/data_cube_inspector.py

```diff
@@ -188,3 +225,71 @@ def get_code_lists_and_cols(self, csv_url: str) -> CodelistsResult:
         """
 
         return self._codelists_and_cols.get(csv_url, CodelistsResult([], 0))
+
+    def get_column_component_info(self, csv_url: str) -> List[ColumnComponentInfo]:
+        """
+        Get the ColumnComponentInfo which contains the EndUserColumnType
+        and the corresponding ColumnDefinition given a csv url
+        """
+
+        list_to_return = []
+        column_definitions = self.csvw_inspector.get_column_definitions_for_csv(csv_url)
+        column_definitions = [x for x in column_definitions if not x.virtual]
```

Comment:
Could you merge this with the last line so we don't do any reassignment like this?